### PR TITLE
Get queue error

### PR DESCRIPTION
**Pull request by theibrahim**
When 0 queued-listings backend was returning error. fixed it. `pages/api/business/listing/getQueue.js`

### CODE REVIEW

File: pages/api/business/listing/getQueue.js [hunk 1]
```diff
@@ -22,10 +22,22 @@ const handler = async (req, res) => {
       },
     },
   });
+
+  if (!queue) {
+    // Handle the case where the queue is not found
+    return res.status(200).json({ listings: [] });
```
Comment (pauloouriques) [inline pages/api/business/listing/getQueue.js:28]: good job

File: pages/api/business/listing/pushQueuedListing.js [hunk 2]
```diff
@@ -59,15 +71,26 @@ const handler = async (req, res) => {
...
+      console.log("New Listing:", newListing);
+
       await prisma.queuedListing.delete({
         where: { id: data.id },
       });
 
-      return newListing;
-    });
+      // Commit the transaction
+      await prisma.$executeRaw`COMMIT`;
```
Comment (pauloouriques) [inline pages/api/business/listing/pushQueuedListing.js:82]: is it fine to mix up raw with regular commands?

**Review by pauloouriques — APPROVED** [review]
lgtm, just had a question regarding mixing up raw with regular commands.

File: pages/api/business/listing/pushQueuedListing.js (hunk 2 above)
Comment (immutable-bee) [reply pages/api/business/listing/pushQueuedListing.js:82]: It should be, provided regular operations are scoped entirely inside raw commands or vice versa

**Pull request closed without merging**